lagunita/lms/static/sass/partials/base/custom_overrides/_mobile.scss

```diff
@@ -39,7 +37,24 @@ section.container {
     }
 }
 
-@media (max-width: $screen_width_small) {
+// Phone screens (Phones only)
```

Comment:
Redundant comment is redundant (that the phone screens are for phones only) ;)

And really, isn't the point of defining t-shirt sizes to avoid mentioning types of devices?
Technically shrinking my browser means I'll use the xs media queries, even though I'm not on a phone...